### ISSUE TITLE
FLF Legend: forever messed up with Empire and Dvaered?

### ISSUE THREAD

**Issue by BM-W1** [naev/naev#2309]
Being an FLF Legend,  Empire and Dvaered planets are hostile and do not allow landing. Improving standings with these factions seems impossible. The only thing that leads to an improvement is finding a derelict spacecraft in a system nearby that is regarded of value for a Dvaered museum (standings 2 points up). 

Using a fake transponder only increases the standing in the fake mode (see attached files) but not in the real identity - which is logic.

Is it intended to have messed up with the Empire and Dvaered permanently or should there be a way of getting a pardon? Rehabilitation programmes for 10M were offered only quite at the beginning of the FLF campaign. They cease to turn up after a certain point.

I did not complete the love letter campaign. Why not make the love letter accessory or the funny dice accessory a prerequisite for being allowed to land on a hostile Empire or Dvaered planet and getting offered a really expensive rehabilitation programme? 

[Archiv.tar.gz](https://github.com/naev/naev/files/11073018/Archiv.tar.gz)

**Comment by not-pyroman** [comment]
yeah, the original creator of that campaign's intention was for it to be impossible to be friendly with the Dvaered or Empire after it is done. Regarding the rehab, the current logic is that if you're above 30 with ANY faction deemed hostile to them, you are not allowed to get a rehab mission.
This campaign is eventually going to get reworked, however ways of increasing it is either pissing off the FLF enough to get rehab missions, or camp and repeatedly kill pirates at one of their hidden jumps

**Comment by bobbens** [comment]
The entire thing needs a rewrite, but you can get out of the negative standing by "farming standing" by killing pirates and such near Empire / Dvaered space. Might be time to finally rethink and rewrite the frontier stuff.

**Comment by ValerioCietto** [comment]
I found it annoying too as it makes impossible to continue the missions with empire and makes traveling very hard because empire and dvaered are the central systems.

I began FLF campaign because at the moment it seems the only way to get to know the hidden jumps needed to pass trough the Incident's Nebula.

**Comment by ValerioCietto** [comment]
Maybe there could be a rehab mission for utter traitors which have more than 20 standing with enemies so that is clearer what to do. 
Or kill missions with no landing requirement so that yoy regain reputation faster.

**Comment by bobbens** [comment]
> Maybe there could be a rehab mission for utter traitors which have more than 20 standing with enemies so that is clearer what to do. Or kill missions with no landing requirement so that yoy regain reputation faster.

That's a good idea for now. Although in case of this particular campaign, it needs quite a rework.
